github/PullRequest.py

```
@@ -489,6 +489,8 @@ def create_review_comment(
         ], subject_type
         assert isinstance(as_suggestion, bool), as_suggestion
 
+        start_side = self.validate_review_comment_start_side(side, start_side, line, start_line, in_reply_to)
+
```

This should fail with 422 if start_side is not set for a multi-line comment.